pkg/apis/autoscaling/v1alpha1/pa_lifecycle.go

```diff
@@ -60,16 +60,16 @@ func (pa *PodAutoscaler) Metric() string {
 	return defaultMetric(pa.Class())
 }
 
-func (pa *PodAutoscaler) annotationInt32(key string) int32 {
+func (pa *PodAutoscaler) annotationInt32(key string) (int32, bool) {
```

Comment:
can we make this method look more like annotationFloat64? I think I would've expected it to be identical except with ParseInt instead of ParseFloat

Reply:
sounds good. i'm keeping the error ignoring though because it's validated in the webhook.

Reply:
well, it may not be - this method is for _any_ annotation, not just the InitialScale one?

Reply:
Yes it's for any annotation, but it's only used my min/max/initialScale. The min/maxScale validation checks for errors and negative values.

Reply:
I guess what Im asking is if we later add another annotation with different validation, won't it be a bit odd/suprising that this method acts differently from annotationFloat64?

Reply:
I guess this isn't something you changed in this PR, this method was already subtly different from the other one 🤷 .

Reply:
Yeah that's true. For consistency, we could change this into not ignoring parsing errors, which won't make a difference in the current behaviors.